### PR TITLE
Add async/await example.

### PR DESCRIPTION
**Pull request by mattcollier**
If this is accurate and of interest, I would like to add an additional function that demonstrates parallelism using `MultiplexedConnection`.

### CODE REVIEW

**Comment by badboy** [comment]
We should extract the example from lib.rs instead: https://github.com/mitsuhiko/redis-rs/blob/master/src/lib.rs#L304-L319

It's more concise and avoids the awful unwraps ;)

**Comment by mattcollier** [comment]
@badboy Done. Thanks for pointing out that code.  Provided this makes it in, should the code in that comment be replaced with a reference to the example?

**Comment by badboy** [comment]
> Provided this makes it in, should the code in that comment be replaced with a reference to the example?

No, having it in lib.rs has the nice advantage of getting rendered on the docs page: https://docs.rs/redis/0.13.0/redis/#async

File: examples/async-await.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+use redis::AsyncCommands;
+
+#[tokio::main]
+async fn main() -> redis::RedisResult<()> {
+  let client = redis::Client::open("redis://127.0.0.1/").unwrap();
```
Comment (badboy) [inline examples/async-await.rs]: 4-spaces indent (or just run `rustfmt` on that file)

Comment (mattcollier) [reply examples/async-await.rs]: Fixed.

**Pull request merged**